vmdb/app/views/layouts/quadicon/_ext_management_system.html.haml

```diff
@@ -1,4 +1,12 @@
-- if @settings[:quadicons][@layout == 'ems_infra' ? :ems : :ems_cloud]
+- db = case @layout
+- when "ems_infra"
+  - :ems
+- when "ems_cloud"
+  - :ems_cloud
+- else
+  - :ems_container
```

Comment:
I love that idea @martinpovolny.  That would make it easier for a novice like me to read.